### PR TITLE
Standardise the names of feature tests

### LINKED ISSUE #221: Standardise all names of tests
Following #212.

### PR DESCRIPTION
**Pull request by allenwq**
Fix #221 

Any further discussion about naming ? 

Should we use <code>'Admin: Announcement: Pagination'</code> or just <code>'Admin: Announcement'</code> ?

### CODE REVIEW

File: spec/features/global_announcements_spec.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe 'Global announcements', type: :feature do
+RSpec.describe 'GlobalAnnouncements', type: :feature do
```
Comment (lowjoel) [inline spec/features/global_announcements_spec.rb]: `Global Announcements` seems correct?

File: spec/features/system_announcements_management_spec.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe 'System announcement management', type: :feature do
+RSpec.describe 'SystemAnnouncement: Management', type: :feature do
```
Comment (lowjoel) [inline spec/features/system_announcements_management_spec.rb]: `System announcements`

**Comment by lowjoel** [comment]
For features I think `Administration: Announcements` would do, the user stories would be in the scenario.

**Comment by allenwq** [comment]
@lowjoel Updated, I'll rename them to feature/scenario in another PR.

**Comment by lowjoel** [comment]
@allenwq needs a rebase

**Pull request merged**